packages/vega5-extension/test/utils.ts

```diff
@@ -14,23 +14,23 @@ interface ISize {
 export const DEFAULT_SIZE: number = 200;
 export const SCALE_FACTOR_PROP: string[] = [
   'metadata',
-  VEGALITE4_MIME_TYPE,
+  VEGALITE5_MIME_TYPE,
   'embed_options',
   'scaleFactor'
 ];
 
 export const VEGALITE4_RENDERER = new RenderedVega({
   latexTypesetter: null,
   linkHandler: null,
-  mimeType: VEGALITE4_MIME_TYPE,
+  mimeType: VEGALITE5_MIME_TYPE,
   resolver: null,
   sanitizer: {
     sanitize: (s: string) => s
   }
 });
 
-export const VEGALITE4_SPEC: JSONObject = {
-  $schema: 'https://vega.github.io/schema/vega-lite/v4.json',
+export const VEGALITE5_SPEC: JSONObject = {
```

Comment:
The tests are now failing as `VEGALITE4_SPEC` is no longer here (https://github.com/jupyterlab/jupyterlab/actions/runs/4046843519/jobs/6960151323). You could either update the tests in `jupyterlab/packages/vega5-extension/test/renderer.spec.ts` or copy-paste the old spec adding a comment stating that v4 is deprecated (if that makes sense, I am not too familiar with vega extension).